### PR TITLE
Set default route to /reminders

### LINKED ISSUE #2: Adjust root to /reminders
Client has changed their mind.

When the user visits the root of the application, they should be redirected to '/reminders' and should see a list of all reminders on the page.

**Hints**:

Dig into the docs for notes on `beforeModel()`.
Maybe look [here](https://guides.emberjs.com/v2.12.0/routing/redirection/).
Your model hook should move out of routes/index.js, and your `router.js` will need adjustments.
Keep in mind that the structure of your `router.js` represents the file structure of the entire app.
**Hot Tip**: This will make your previous test fail. Make the necessary changes in the test(s).

### PR DESCRIPTION
**Pull request by ActionJonny**
## Purpose

Updated the root to redirect you to /reminders.

## Approach

Generated an Index route and used beforeModel to redirect the users default route to be /reminders.

### Learning

Used ember docs to find information on the beforeModel method

### Test coverage 

We updated our first test to reflect the changes to root

### Merge Dependencies and Related Work

closes #2

### Follow-up tasks

- Issue #3 

### Screenshots

http://recordit.co/2ZcPUohDJw

### CODE REVIEW

**Comment by ActionJonny** [comment]
@Tman22 @martensonbj, We have a pull request up to close issue #2

File: tests/unit/routes/index-test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:index', 'Unit | Route | index', {
```
Comment (Tman22) [inline tests/unit/routes/index-test.js]: Take this test out

Comment (martensonbj) [reply tests/unit/routes/index-test.js]: Status on this? @jonwille @hmorri32

**Comment by ActionJonny** [comment]
@Tman22 @martensonbj, We removed the code and deleted the tests/unit/routes/index-test.js file

**Comment by martensonbj** [comment]
👍

**Pull request merged**